#!/bin/bash 
#SBATCH -D /clusterfs/rosalind/users/makman/GATK/sams/
#SBATCH -J map_10
#SBATCH --account=co_rosalind
#SBATCH --partition=savio2_htc
#SBATCH --qos=rosalind_htc2_normal
#SBATCH --ntasks=1
#SBATCH --cpus-per-task=6
#SBATCH --time=480:00:00
#SBATCH -o /global/scratch/makman/GATK/sams//global/home/users/makman/GATK/outs/map10.out
#SBATCH -e /global/home/users/makman/GATK/outs/map10.err
#SBATCH --mail-user=makman@berkeley.edu
#SBATCH --mail-type=All

# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Anzac_Pueblo_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Anzac_Pueblo_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Anzac_Pueblo.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Arikara_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Arikara_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Arikara.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/HA124_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/HA124_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/HA124.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/HA316_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/HA316_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/HA316.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/HA404_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/HA404_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/HA404.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/HA433_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/HA433_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/HA433.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/HA442_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/HA442_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/HA442.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/HA821_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/HA821_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/HA821.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/HA89_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/HA89_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/HA89.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Havasupai_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Havasupai_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Havasupai.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Hidatsa1_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Hidatsa1_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Hidatsa1.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Hopi_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Hopi_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Hopi.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Hopi_dye_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Hopi_dye_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Hopi_dye.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Hopi_mapping_parent_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Hopi_mapping_parent_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Hopi_mapping_parent.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Hopi_other_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Hopi_other_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Hopi_other.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Laguna_Pueblo_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Laguna_Pueblo_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Laguna_Pueblo.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Mandan1_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Mandan1_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Mandan1.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Mandan2_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Mandan2_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Mandan2.sam
/clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/MexCult14_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/MexCult14_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/MexCult14.sam
/clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/MexCult15_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/MexCult15_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/MexCult15.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/MexCult1_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/MexCult1_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/MexCult1.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/MexCult3_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/MexCult3_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/MexCult3.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/MexCult6_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/MexCult6_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/MexCult6.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/MexCult7_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/MexCult7_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/MexCult7.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/MexCult9_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/MexCult9_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/MexCult9.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/PPN255_Manchurian_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/PPN255_Manchurian_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/PPN255_Manchurian.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/PPN262_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/PPN262_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/PPN262.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Pueblo_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Pueblo_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Pueblo.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/RHA271_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/RHA271_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/RHA271.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/RHA309_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/RHA309_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/RHA309.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/RHA311_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/RHA311_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/RHA311.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/RHA328_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/RHA328_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/RHA328.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/RHA355_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/RHA355_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/RHA355.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/RHA358_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/RHA358_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/RHA358.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/RHA408_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/RHA408_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/RHA408.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/RHA426_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/RHA426_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/RHA426.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/RHA857_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/RHA857_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/RHA857.fastq.gz
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/SF33_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/SF33_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/SF33.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Seneca_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Seneca_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Seneca.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/VIR847_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/VIR847_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/VIR847.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/Zuni_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/Zuni_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/Zuni.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/ann06-seWY_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/ann06-seWY_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/ann06-seWY.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/ann08-ceTN_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/ann08-ceTN_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/ann08-ceTN.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/ann1238_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/ann1238_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/ann1238.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/ann34-cwKS_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/ann34-cwKS_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/ann34-cwKS.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/ann39-ncNE_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/ann39-ncNE_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/ann39-ncNE.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annIA_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annIA_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annIA.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annKS2W-35_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annKS2W-35_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annKS2W-35.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annKS_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annKS_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annKS.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annMB1W-40_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annMB1W-40_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annMB1W-40.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annMO1W-39_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annMO1W-39_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annMO1W-39.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annMex_A1516_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annMex_A1516_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annMex_A1516.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annMex_A1517_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annMex_A1517_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annMex_A1517.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annMex_A1572_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annMex_A1572_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annMex_A1572.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annMex_A1574_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annMex_A1574_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annMex_A1574.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annMex_Ann261_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annMex_Ann261_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annMex_Ann261.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annND1W-6_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annND1W-6_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annND1W-6.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annNM_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annNM_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annNM.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annSD1W-35_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annSD1W-35_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annSD1W-35.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annSD2W-18_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annSD2W-18_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annSD2W-18.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annSK1W-Q_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annSK1W-Q_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annSK1W-Q.sam
# /clusterfs/rosalind/users/makman/NextGenMap-0.5.2/bin/ngm-0.5.2/ngm -r /clusterfs/rosalind/users/makman/HanXRQr2/HanXRQr2.0-SUNRISE-2.1.genome.fasta -p -1 ../fastq/raw/ready/annWY_R1_trimmed.fastq.gz -2 ../fastq/raw/ready/annWY_R2_trimmed.fastq.gz -o /global/scratch/makman/GATK/sams/annWY.sam
